app/controllers/api/stages_controller.rb

```diff
@@ -18,10 +16,19 @@ def clone
     end
   end
 
+  def duplicable
+    render json: @project.duplicable_stage
+  end
+
+  def put_duplicable
```

Comment:
`create_duplicate` ... how is this different from `clone` ... ? ... more docs :D